### DISCUSSION TITLE
Feature request: naming of backup files by provided filename template

### DISCUSSION THREAD

**Discussion by iamjwk1** [lavinir/hassio-onedrive-backup#180] [Ideas]
For easier recognition of backup files and sorting the files chronologically it would be very handy if users can define a backup filename based on a template provided, which includes day, month, year, hour and minutes next to fixed text + if it is a full backup or hourly backup.

So instead of for example:
DailyBackup_ Sunday, February 11, 2024.tar
HourlyBackup_ Sunday, 3_00 PM, February 11, 2024.tar

I get for example:
20240211-0250-Sun Daily backup Home Assistant 1.tar
20240211-1500-Sun Hourly backup Home Assistant 1.tar

Regards,
Jan-Willem

**Comment by iamjwk1** [comment]
Hmm stupid.. I can define names via autobackup blueprint I just found out...

**Reply by lloyda** [reply]
Can you explain this?  Don't understand which blueprint you are referring to (and don't recognise the format of your example files either).
